cv_next/app/cv/[cvId]/components/commentSection/comment.tsx

```diff
@@ -0,0 +1,118 @@
+"use client";
+import { deleteComment } from "@/app/actions/comments/deleteComment";
+import { setResolved } from "@/app/actions/comments/setResolved";
+import { upvoteComment } from "@/app/actions/comments/setLike";
+import useSWR, { mutate } from "swr";
+import { getUserModel } from "@/app/actions/users/getUser";
+
+export default function Comment({
```

Comment:
There are many non-memoized objects here.. I think you should check re-render effects here, and see if memoization is needed.

Reply:
Managed to get re-renders down using innerjoin on the select queries for the username